roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -26,7 +26,11 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 # to be loaded from the neutron.ml2.mechanism_drivers namespace.
 # Example: mechanism_drivers = arista
 # Example: mechanism_drivers = cisco,logger
-mechanism_drivers = linuxbridge
+{% set mechanism_drivers = ['linuxbridge'] %}
+{% if neutron.l2_population %}
+{% set _ = mechanism_drivers.append('l2population') %}
```

Comment:
I missed this in the previous PR. The set _, is that special in jinja for a way to reference oneself when doing a set?